### DISCUSSION TITLE
Discussions, for the tray icons Style for OpenFreezeCenter. Let's start!!!

### DISCUSSION THREAD

**Discussion by Special-Niewbie** [YoCodingMonster/OpenFreezeCenter#21] [Ideas]
In mean time @YoCodingMonster and some user expert on coding will help on this amazing project, why we don't start to give some ideas about the looking for this legendary Linux Software!!!

I don't know if @YoCodingMonster , already thought about it, and it's not to overpass him on his thoughts, but to have some idea all together.
This, to stimulate the topic, and I'm dropping one idea about the minimal icon tray looking in meanwhile the app it's running in back ground . I was thinking to mix up a **Linux Penguin** with the **MSI Dragon Based**  in minimal style as the examples below:

Penguin + Dragon:

![Penguin_Dragon_3](https://user-images.githubusercontent.com/64843123/120482311-96cde300-c3e3-11eb-9563-4dfaf57597fd.jpg)

And make it on tray Icon Minimal as the examples below:

![Examples](https://user-images.githubusercontent.com/64843123/120482413-b238ee00-c3e3-11eb-9227-8b55600799d5.png)

What do you think? Other Ideas?

**Comment by dydzio0614** [comment]
penguin + dragon is awesome

**Comment by Special-Niewbie** [comment]
To whom it's interested I have extracted the original MSI icon (in different resolutions) from the original .exe file, to apply on OpenFreezeCenter:

![Schermata del 2021-06-20 13-16-41](https://user-images.githubusercontent.com/64843123/122663083-8f238200-d1ca-11eb-8e03-9c611221bca5.png)

I have made also the Penguin Dragon Icon:

![Penguin2](https://user-images.githubusercontent.com/64843123/122663234-7798c900-d1cb-11eb-957c-08a7c04f8d97.png)

**Reply by YoCodingMonster** [reply]
The penguin dragon looks cute...
Amazing work...

Soo happy to see people collaborate

**Reply by IvanoiuAlexandruPaul** [reply]
Hello. Since I think you are Italian and maybe you know more than me and know how to install the program on Linux can you give me a hand and install the program on my PC too ? Thank you Special Niewbie

**Reply by Special-Niewbie** [reply]
> Hello. Since I think you are Italian and maybe you know more than me and know how to install the program on Linux can you give me a hand and install the program on my PC too ? Thank you Special Niewbie

Too much compliments @IvanoiuAlexandruPaul  ;) thank you, sure let me know what the problems are I will try......

@YoCodingMonster Where I can upload the files as more choices? I need to upload on my Github Account and paste here the address?

**Reply by YoCodingMonster** [reply]
> > Hello. Since I think you are Italian and maybe you know more than me and know how to install the program on Linux can you give me a hand and install the program on my PC too ? Thank you Special Niewbie
> 
> Too much compliments @IvanoiuAlexandruPaul  ;) thank you, sure let me know what the problems are I will try......
> 
> @YoCodingMonster Where I can upload the files as more choices? I need to upload on my Github Account and paste here the address?
> 

You are most welcome to fork this project and work on it....and then keep it as is and paster the link here....
Or
You can merge again with the same project....

As you wish👍🏻

**Reply by IvanoiuAlexandruPaul** [reply]
@Special this is my email : ivalexandrupaul@gmail.com let me know when you can talk send me an email so we can chat thank you

**Reply by Special-Niewbie** [reply]
Thank you so much @YoCodingMonster , I didn't fork till now to respect you and your job on it, but if you tell me this I will start to fork and test it for you because I need to marge the fix why it doesn't display anything at the beginning, if I can be helpfully to test some new coding from your side; I will be happy to help on and test on my laptop.... Do you think I can make these tests with the live USB or better with the esternal SSD tester?

**Reply by Special-Niewbie** [reply]
@IvanoiuAlexandruPaul , do you have Discord to simplify the chatting? Otherwise we can open a new Topic here, like FAQ Sections...let me know.

**Reply by YoCodingMonster** [reply]
> Thank you so much @YoCodingMonster , I didn't fork till now to respect you and your job on it, but if you tell me this I will start to fork and test it for you because I need to marge the fix why it doesn't display anything at the beginning, if I can be helpfully to test some new coding from your side; I will be happy to help on and test on my laptop.... Do you think I can make these tests with the live USB or better with the external SSD tester?

thanks for testing and doing all this..respect +

installing a full copy of ubuntu on external SSD will work because the application needs to reboot to make some changes and live UBS resets itself at reboot... so Installing ubuntu on External SSD works.

i Myself use this. i had 2 SSD in my laptop [windows 10 + UBUNTU], and the test for other distros was done by installing the OS completely on an External NVMe SSD and doing testing from that.

**Reply by Special-Niewbie** [reply]
Ok Sure..... because in my laptop I have 4 OS installed Win 7, 2 Win10 and Ubuntu....but before I use Linux always by external SSD/USB , now I have shifted everything for internal ....anyway i will prepare the external Ubuntu, and when the software will be more stable I can test other Distros like Solus o Debian or anything else to test with various OS how the app it's performing.....

**Reply by IvanoiuAlexandruPaul** [reply]
@Special-Niewbie I am online if you are available. let me know but writing me on Discord. My name is IvanoiuPaul#7483 . Thank you. Waiting for a sign.

**Comment by IvanoiuAlexandruPaul** [comment]
@Special this is my Discord name : IvanoiuPaul#7483 in 10 min I’m online
